frontend/packages/shared/src/api/queries.ts

```diff
@@ -133,6 +134,8 @@ export const getOptionList = (owner: string, app: string, optionsListId: string)
 export const getOptionLists = (owner: string, app: string) => get<OptionListsResponse>(optionListsPath(owner, app));
 export const getOptionListsReferences = (owner: string, app: string) => get<OptionListReferences>(optionListReferencesPath(owner, app));
 export const getOptionListIds = (owner: string, app: string) => get<string[]>(optionListIdsPath(owner, app));
+export const getAvailableOptionListDataListsInOrg = (owner: string): Promise<string[]> => Promise.resolve(listOfAvailableOptionListTitlesToImport); // Todo: Replace with real API call when endpoint is ready. https://github.com/Altinn/altinn-studio/issues/14885
```

Comment:
Since we are only fetching the option list titles, what do you think about renaming to:
```suggestion
export const getAvailableOptionListsInOrg = (owner: string): Promise<string[]> => Promise.resolve(listOfAvailableOptionListTitlesToImport); // Todo: Replace with real API call when endpoint is ready. https://github.com/Altinn/altinn-studio/issues/14885
```

Reply:
I agree, but I believe we maybe should include the titles as well 😄 Maybe `getAvailableOptionListTitlesInOrg`?

Reply:
Yes 😊